ksqldb-functional-tests/src/test/resources/query-validation-tests/replace.json

```diff
@@ -23,6 +23,23 @@
         {"topic": "OUTPUT", "value": {"REPLACE":null, "REPLACE_NULL":null, "REPLACE_W_NULL":null}},
         {"topic": "OUTPUT", "value": {"REPLACE":"popcorn", "REPLACE_NULL":null, "REPLACE_W_NULL":null}}
       ]
+    },
+    {
+      "name": "regexp_replace",
+      "statements": [
+        "CREATE STREAM TEST (K STRING KEY, input_string VARCHAR) WITH (kafka_topic='test_topic', value_format='JSON');",
+        "CREATE STREAM OUTPUT AS SELECT K, REGEXP_REPLACE(input_string, '(ab)+', 'cat') AS EXTRACTED FROM TEST;"
+      ],
+      "inputs": [
+        {"topic": "test_topic", "value": {"input_string": "baababaa"}},
+        {"topic": "test_topic", "value": {"input_string": "baabbabaa"}},
+        {"topic": "test_topic", "value": {"input_string": null}}
+      ],
+      "outputs": [
+        {"topic": "OUTPUT", "value": {"EXTRACTED": "bacataa"}},
+        {"topic": "OUTPUT", "value": {"EXTRACTED": "bacatbcataa"}},
+        {"topic": "OUTPUT", "value": {"EXTRACTED": null}}
+      ]
```

Comment:
Would be good if this test case covered nulls being passed to all params.

Reply:
Done